src/pspm_cfg/pspm_cfg_pp_heart_data.m

```diff
@@ -66,85 +66,74 @@
 ecg2hb_amri_signal_to_use.values  = {'ecg', 'teo', 'auto'};
 ecg2hb_amri_signal_to_use.labels  = {'Filtered ECG signal', 'Filtered and TEO applied ECG signal',...
     'Choose automatically based on autocorrelation'};
-ecg2hb_amri_signal_to_use.help    = {['Which signal to feed to the core heartbeat detection procedure. ',...
-    '''ecg'' corresponds to filtered ECG signal. ''teo'' corresponds to the signal obtained by filtering ',...
-    'the ECG signal even more and applying the Teager Enery Operator. ''auto'' option picks the one of ',...
-    'these two options that results in higher autocorrelation']};
+ecg2hb_amri_signal_to_use.help    = pspm_cfg_help_format('pspm_convert_ecg2hb_amri', 'options.signal_to_use');
 
 ecg2hb_amri_hrrange         = cfg_entry;
 ecg2hb_amri_hrrange.name    = 'Feasible heartrate range';
 ecg2hb_amri_hrrange.tag     = 'hrrange';
 ecg2hb_amri_hrrange.strtype = 'r';
 ecg2hb_amri_hrrange.num     = [1 2];
 ecg2hb_amri_hrrange.val     = {[20 200]};
-ecg2hb_amri_hrrange.help    = {'Define the minimum and maximum possible heartrates for your data'};
+ecg2hb_amri_hrrange.help    = pspm_cfg_help_format('pspm_convert_ecg2hb_amri', 'options.hrrange');
 
 ecg2hb_amri_ecg_bandpass         = cfg_entry;
 ecg2hb_amri_ecg_bandpass.name    = 'ECG bandpass filter';
 ecg2hb_amri_ecg_bandpass.tag     = 'ecg_bandpass';
 ecg2hb_amri_ecg_bandpass.strtype = 'r';
 ecg2hb_amri_ecg_bandpass.num     = [1 2];
 ecg2hb_amri_ecg_bandpass.val     = {[0.5 40]};
-ecg2hb_amri_ecg_bandpass.help    = {'Define the cutoff frequencies (Hz) for bandpass filter applied to raw ECG signal'};
+ecg2hb_amri_ecg_bandpass.help    = pspm_cfg_help_format('pspm_convert_ecg2hb_amri', 'options.ecg_bandpass');
 
 ecg2hb_amri_teo_bandpass         = cfg_entry;
 ecg2hb_amri_teo_bandpass.name    = 'TEO bandpass filter';
 ecg2hb_amri_teo_bandpass.tag     = 'teo_bandpass';
 ecg2hb_amri_teo_bandpass.strtype = 'r';
 ecg2hb_amri_teo_bandpass.num     = [1 2];
 ecg2hb_amri_teo_bandpass.val     = {[8 40]};
-ecg2hb_amri_teo_bandpass.help    = {['Define the cutoff frequencies (Hz) for bandpass filter applied to filtered ECG',...
-    ' signal before applying TEO']};
+ecg2hb_amri_teo_bandpass.help    = pspm_cfg_help_format('pspm_convert_ecg2hb_amri', 'options.teo_bandpass');
```

Comment:
Just now sure if this is expected...

<img width="860" alt="image" src="https://github.com/user-attachments/assets/6a70a816-514a-4736-aad6-52bf9e814de7">

Reply:
fixed.